frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -359,21 +435,40 @@ function ReaderFooter:updateFooterTextGenerator()
     return true
 end
 
-local option_titles = {
-    all_at_once = _("Show all at once"),
-    reclaim_height = _("Reclaim bar height from bottom margin"),
-    toc_markers = _("Show chapter markers"),
-    page_progress = _("Current page"),
-    time = _("Current time"),
-    pages_left = _("Pages left in chapter"),
-    battery = _("Battery status"),
-    percentage = _("Progress percentage"),
-    book_time_to_read = _("Book time to read"),
-    chapter_time_to_read = _("Chapter time to read"),
-    frontlight = _("Frontlight level"),
-    mem_usage = _("KOReader memory usage"),
-    wifi_status = _("Wi-Fi status"),
-}
+function ReaderFooter:progressPercentage(digits)
+    local symbol_type = self.settings.text_prefix or "icons"
+    local prefix = symbol_prefix[symbol_type].percentage
+
+    local string_percentage
+    if not prefix then
+        string_percentage = "%." .. digits .. "f%%"
+    else
+        string_percentage = prefix .. " %." .. digits .. "f%%"
+    end
+    return string_percentage:format(self.progress_bar.percentage * 100)
+end
+
+function ReaderFooter:textOptionTitles(option)
+    local symbol = self.settings.text_prefix or "icons"
+    local option_titles = {
+        all_at_once = _("Show all at once"),
+        reclaim_height = _("Reclaim bar height from bottom margin"),
+        toc_markers = _("Show chapter markers"),
+        page_progress = T(_("Current page (%1)"), "/"),
+        time = symbol_prefix[symbol].time
+            and T(_("Current time (%1)"), symbol_prefix[symbol].time) or _("Current time"),
+        pages_left = T(_("Pages left in chapter (%1)"), symbol_prefix[symbol].pages_left),
+        battery = T(_("Battery status (%1)"), symbol_prefix[symbol].battery),
+        percentage = symbol_prefix[symbol].percentage
+            and T(_("Progress percentage (%1)"), symbol_prefix[symbol].percentage) or ("Progress percentage"),
+        book_time_to_read = T(_("Book time to read (%1)"),symbol_prefix[symbol].book_time_to_read),
+        chapter_time_to_read = T(_("Chapter time to read (%1)"), symbol_prefix[symbol].chapter_time_to_read),
+        frontlight = T(_("Frontlight level (%1)"), symbol_prefix[symbol].frontlight),
+        mem_usage = T(_("KOReader memory usage (%1)"), symbol_prefix[symbol].mem_usage),
+        wifi_status = T(_("Wi-Fi status (%1)"), symbol_prefix[symbol].wifi_status),
+    }
+    return option_titles[option]
+end
 
 function ReaderFooter:addToMainMenu(menu_items)
```

Comment:
That menu is getting crowded :) I wouldn't mind a separator between the global options and the list of available footer items (or have these in a sub menu?).
Also may be move the `Settings>` among the top ones instead of having it burried after the list of footer items?

Reply:
Done. Settings moved to top.